.travis.yml

```diff
@@ -24,7 +24,7 @@ before_script:
 jobs:
   include:
     - stage: test
-      script: jabba use $JDK  && java -version && sbt ++$TRAVIS_SCALA_VERSION test
+      script: jabba use $JDK  && java -version && sbt ++$TRAVIS_SCALA_VERSION "scalafmtCheck;test"
```

Comment:
it's even `scalafmtCheckAll` to include test sources

Reply:
Ah thanks!